helm_deploy/calculate-journey-variable-payments/templates/_envs.tpl

```diff
@@ -118,6 +118,9 @@ env:
   - name: CRON_IMPORT_REPORTS
     value: "{{ .Values.env.CRON_IMPORT_REPORTS }}"
 
+  - name: CRON_AUTOMATIC_LOCATION_MAPPING
+    value: "{{ .Values.env.CRON_AUTOMATIC_LOCATION_MAPPING }}"
```

Comment:
Should this be `CRON_BACKFILL_REPORTS`?

Reply:
Ah yes it should.  Copy and paste failure.